propolis/src/dispatch/async_tasks.rs

```diff
@@ -50,7 +51,7 @@ impl Inner {
         let mut ctrls = self.ctrls.lock().unwrap();
 
         let id = state.next_id;
-        state.next_id += 1;
+        state.next_id.0 += 1;
```

Comment:
Hmm, could we impl Add and/or AddAssign on CtxId to avoid reaching into the tuple?

Reply:
That would let anyone with a `CtxId` to manufacture a different one. The adding is just an implementation detail anyways and doesn't really make sense as something to lift to the type itself.

Reply:
SG